src/Forms/Main.cs

```diff
@@ -6625,11 +6618,7 @@ private void ButtonAutoBreakClick(object sender, EventArgs e)
 
         private static void FixVerticalScrollBars(TextBox tb)
         {
-            var lineCount = Utilities.CountTagInText(tb.Text, Environment.NewLine) + 1;
-            if (lineCount > 3)
-                tb.ScrollBars = ScrollBars.Vertical;
-            else
-                tb.ScrollBars = ScrollBars.None;
+            tb.ScrollBars = Utilities.CountTagInText(tb.Text, Environment.NewLine) + 1 > 3 ? ScrollBars.Vertical : ScrollBars.None;

```

Comment:
Seems a bit hard to read - also the new Utilities.GetNumberOfLines is probably better?